bigtable/client/testing/table_integration_test.cc

```diff
@@ -89,6 +89,20 @@ void TableIntegrationTest::CreateCells(
   table.BulkApply(std::move(bulk));
 }
 
+std::vector<bigtable::Cell> TableIntegrationTest::GetCellsIgnoringTimestamp(
+    std::vector<bigtable::Cell> cells) {
+  // Create the expected_cells and actual_cells with same timestamp
+  std::vector<bigtable::Cell> return_cells;
+  for (auto& cell : cells) {
```

Comment:
I think this could be replaced with a `std::transform()` call, but maybe I am trying to be too clever, e.g.:

```C++
std::transform(cells.begin(), cells.end(), std::back_inserter(return_cells), [](Cell const& c) { ... });
```

Reply:
Is there any shorter solution than this for using std::transform

  std::transform(cells.begin(),cells.end(), std::back_inserter(return_cells),
		  [](Cell const& cell){ bigtable::Cell newCell(cell.row_key(), cell.family_name(),
                  cell.column_qualifier(), 0, cell.value(), cell.labels()) ; return newCell ;});